ee/tables/jwt/table.go

```diff
@@ -0,0 +1,176 @@
+package kolide_jwt
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+var (
+	ErrMissingKeyId     = errors.New("no key id found in the JWT header")
+	ErrMatchingKeyId    = errors.New("no key id matched the JWT header key id")
+	ErrParsingPemBlock  = errors.New("error parsing PEM block containing the public key")
+	ErrParsingPublicKey = errors.New("error parsing the public key from the PEM block")
+)
+
+type Table struct {
+	slogger *slog.Logger
+}
+
+func TablePlugin(slogger *slog.Logger) *table.Plugin {
+	columns := dataflattentable.Columns(
+		table.TextColumn("path"),
+		table.TextColumn("signature_key"),
+	)
+
+	t := &Table{
+		slogger: slogger.With("table", "kolide_jwt"),
+	}
+
+	return table.NewPlugin("kolide_jwt", columns, t.generate)
+}
+
+func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	paths := tablehelpers.GetConstraints(queryContext, "path")
+	if len(paths) < 1 {
+		return nil, fmt.Errorf("kolide_jwt requires at least one path to be specified")
+	}
```

Comment:
Prefer this block be near the `for _, path := range paths` loop.

Reply:
Okay, I'll move it.